.devcontainer/on-create.sh

```diff
@@ -2,6 +2,16 @@
 
 echo "on-create started" >> $HOME/status
 
+# Install k3d > 5.0.1
+k3d --version | grep -Eo '^k3d version v5...[1-9]$' > /dev/null 2>&1
+if [ $? -ne 0 ]; then
+    # Means we don't have proper k3d version
+    # Install v5.0.1
+    echo "Installing k3d v5.0.1"
```

Comment:
Ideally k3d binary should be included in the docker image.
Once, the docker image have k3d included, we will remove k3d installation here.
Since it doesn't have that yet, I've just included the version which is known to work.

@aflinchb  I'll change the installation to `wget -q -O - https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash` which will install latest k3d.